### ISSUE TITLE
Redundant rendering on curation page

### ISSUE THREAD

**Issue by reckart** [webanno/webanno#778]
The curation page (or probably rather the curation panel) requests the rendering of the annotation editor (in particular the "mergeview") by multiple means causing a JavaScript error message in the client:

```
TypeError: undefined is not an object (evaluating 'Wicket.$('vis5d').dispatcher.post')
(anonymous function) — Anonymous Script 2 (line 1:8642)
```

It looks like both, `AnnotationEditorBase.requestRender()` and `target.add(mergeview)` are used. 

Not a critical problem - the UI still seems to work nicely - but it is definitely unnecessary.

**Comment by reckart** [comment]
This should be superseded by the fix for #418.

**Comment by reckart** [comment]
Cannot reproduce anymore. I assume it was fixed by #418.

WebAnno -- 3.4.5-SNAPSHOT (2018-07-08 21:33:52, build b483c04774de4623400014802c7799decf83b333)
